src/allmydata/web/status.py

```diff
@@ -1164,82 +1172,133 @@ def render_upload_bytes_fetched(self, ctx, data):
     def render_upload_bytes_encoded(self, ctx, data):
         return str(data["chk_upload_helper.encoded_bytes"])
 
+# ------------------------------------------------------------------------
+
```

Comment:
Don't think we do this anywhere else in Tahoe?

Reply:
`web/status.py` renders several pages: status and friends (upload status, publish status, retrieve status, map update status. download status), helper status, and statistics.  It made sense to add some kind of visual separator between the classes that are responsible for different pages.  

I will remove it if inconsistency is a problem.

Reply:
Removed in 6c3806c.